content/root/best-practice-guide.textile

```diff
@@ -0,0 +1,146 @@
+---
+title: Best Practice Guide
+section: root
+index: 30
+jump_to:
+  Contents:
+    - Ably 101
+    - Using Ably like a pro
+---
+
+h2(#title). Ably Best Practice Guide
+
+This best practice guide aims to quickly get you to understand the best way to make use of Ably's Realtime platform for your applications. These recommendations include some important considerations that are otherwise prone to easy ignoration while skimming through the docs. 
+
+This guide consists of the following two sections:
+* Ably 101 - Key concepts that you must understand before moving on to the best practice recommendations.
+* Using Ably like a pro - Actual recommendations that we've compiled as per our experience with our users, in order for you to avoid common mistakes or misunderstandings whilst using Ably.
+
+h3(#ably-101). Ably 101
+
+Ably Realtime is a cloud-based platform that allows you to easily integrate realtime functionality into your applications. Just plug and play.
+
+h4(#pubsub). Publish and Subscribe (Pub/Sub)
+
+You can publish data to a named "channel":/realtime/channels-messages on the Ably Realtime platform, and all the clients subscribed to that very channel will receive this data in realtime, as it is being published.
+
+h4(#notifications). Notifications 
+
+You can use "Push Notifications":/realtime/push to send notifications to your clients even when they are currently not using your application, or in other words, not connected to Ably. You can do this on mobile devices as well as browsers. You can also use "Smart Notifications":/general/smart-notifications for your notifications to be most effective depending on your user's connection statte, their active device or even their current context within your application.
+
+h4(#presence). Presence
+
+You can have your clients attach to the presence channel using "Presence":/realtime/presence in order to indicate if these clients (or devices in case of IoT clients) are online or connected. The changes in this connection status, if any, can also be tracked by optionally having your clients subscribe to the same presence channel.
```

Comment:
"You can have your clients attach to the presence channel using "Presence":/realtime/presence in order to indicate if these clients (or devices in case of IoT clients) are online or connected." is confusing.

Try:

"Your clients can attach to a presence channel using "Presence":/realtime/presence and announce they are present on a channel.  This is a useful for example in chat apps to indicate a member is online, or with IoT devices to indicate a device is online and connected. Clients can subscribe to realtime presence changes on channels, such as members entering or leaving, and respond to these events."